common/build/eslint-config-fluid/recommended.js

```diff
@@ -56,6 +56,11 @@ module.exports = {
 		 */
 		"unicorn/no-nested-ternary": "off",
 
+		/**
+		 * Disabled due to false positives / disruptive behavior of auto-fix.
```

Comment:
Might be worth mentioning the relevant issue in code and/or that if there's a rule analog which has type information, it's probably good to enable:

https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2018

Reply:
```suggestion
		 * Disabled due to false positives / disruptive behavior of auto-fix.
		 * See {@link https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2018}.
		 * We may consider re-enabling once the above issue has been resolved.
```